content/docs/command-reference/exp/pull.md

```diff
@@ -24,9 +24,12 @@ experiments across <abbr>repository</abbr> copies via Git (and DVC) remotes.
 > these are saved under custom Git references. See **How does DVC track
 > experiments?** in `dvc exp run` to learn more about DVC experiment storage.
 
-A working `git_remote` name (e.g. `origin`) or valid Git repo's URL is required,
+A working `git_remote` name (e.g. `origin`) or valid [SSH Git URL] is required,
 as well as an `experiment` name or hash (see `dvc exp run`) to pull.
 
+[ssh git url]:
+  https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_protocols
```

Comment:
Isn't an SSH Git URL in the form 'git@github.com:iterative/dvc.git'?
and `https://github.com/iterative/dvc.git` is an HTTPS Git URL

Reply:
This is a link to the git documentation regarding supported protocols and URLs (it's the markdown syntax for reference links, so the `[SSH Git URL]` in the previous paragraph will be a link w/this URL)